### PR TITLE
Exclude nodes from external load balancers in Kubernetes 1.19

### PR DESCRIPTION
**Pull request by sarahhodne**
Starting in Kubernetes 1.19, unschedulable nodes were included in load balancers again (see kubernetes/kubernetes#90823), which causes a conflict between the cloud controller and lifecycle-manager, where lifecycle-manager attempts to remove the instance from the load balancer, but then the cloud controller attempts to re-add it.

Kubernetes 1.19 also moved the `ServiceNodeExclusion` feature gate to beta support, and defaults it to enabled, so you can set the `node.kubernetes.io/exclude-from-external-load-balancers` label on the node to have the node be excluded. An older version of this feature used the `alpha.service-controller.kubernetes.io/exclude-balancer`, but this label is removed in Kubernetes 1.19.

This commit sets the new `node.kubernetes.io/exclude-from-external-load-balancers` label in addition to the older alpha label so lifecycle-manager continues to work on both old and new clusters.

### CODE REVIEW

**Comment by sarahhodne** [comment]
CLA assistant was confusing, "Sign in" apparently means you sign the CLA immediately. I did not mean to sign it yet, since I believe I need to have it signed by my workplace, so please hold off on this for now.

**Comment by eytan-avisror** [comment]
Thanks @sarahhodne 
We dont use CLA anymore actually, we need a DCO - so you just need to sign your commits accordingly 
See here: https://github.com/keikoproj/keiko#news

I will remove the CLA assistant from this repo (I guess we left it)

**Comment by eytan-avisror** [comment]
Hey @sarahhodne looks like you may need to recreate this PR since it's out-of-date with base (not sure why it's not allowing a rebase on the PR).

We'd ideally like to merge this soon, let me know if you can use the DCO instead of CLA

**Comment by sarahhodne** [comment]
I'm just checking with legal at $work regarding the DCO, I think that should be fine but I just want their sign-off first. Once I have that, I'll rebase or recreate this PR with the sign-off in the commit.

**Comment by eytan-avisror** [comment]
> I'm just checking with legal at $work regarding the DCO, I think that should be fine but I just want their sign-off first. Once I have that, I'll rebase or recreate this PR with the sign-off in the commit.

OK, sounds good, thanks!

**Comment by sarahhodne** [comment]
@eytan-avisror Added the DCO signoff to my commit and rebased against latest `master` so I think this should be ready for a review now.

**Review by eytan-avisror — APPROVED** [review]
👍🏻

**Pull request merged**
